pkgs/applications/blockchains/lightning-terminal/default.nix

```diff
@@ -0,0 +1,76 @@
+{ pkgs ? import <nixpkgs> { system = builtins.currentSystem; },
+  lib ? pkgs.lib,
+  fetchFromGitHub ? pkgs.fetchFromGitHub,
+  fetchYarnDeps ? pkgs.fetchYarnDeps,
+  nodejs ? pkgs.nodejs,
+  yarn ? pkgs.yarn,
+  gnumake ? pkgs.gnumake,
+  go ? pkgs.go,
+  git ? pkgs.git,
+  stdenv ? pkgs.stdenv,
+  mkYarnPackage ? pkgs.mkYarnPackage,
+  fixup_yarn_lock ? pkgs.fixup_yarn_lock,
+  buildGoModule ? pkgs.buildGoModule
+}:
```

Comment:
Is there a reason to set them like this? For most package, we can just import by name and not have to check and qualify

Reply:
Fixed, thank you